components/supervisor/frontend/src/index.ts

```diff
@@ -64,7 +64,7 @@ const ideService = IDEFrontendService.create();
 const loadingIDE = new Promise((resolve) => window.addEventListener("DOMContentLoaded", resolve, { once: true }));
 const toStop = new DisposableCollection();
 
-document.body.style.visibility = "hidden";
```

Comment:
@filiptronicek why do we do it?

Reply:
@akosyakov I'm trying to get the opening working in Safari. It doesn't respect visibility hidden on the iFrame so trying with opacity.

Reply:
why does it work for VS Code Web?

Reply:
It looks like it's a bug / just different behavior in Safari. It comes down to the order of the elements -> the iframe needs to be first, and that presumably happens because of VS Code loading in the elements asynchronously and xterm having the container element right there from the start.

Reply:
Fixed and PR description updated